community/how_to_ask_on_discourse.md

```diff
@@ -1,4 +1,140 @@
+
+**Your example should demonstrate your problem**
+
+After streamlining your code down to just the essentials, make sure that you double-check that your example code still produces your problem.  If not, perhaps you accidentally solved your own problem!
+
+
+(ask_discourse/title)=
+## A Good Title
+The title of your topic is critical because it's the first thing people will see when you submit your question.  If the question doesn't seem relevant or interesting or if people can't understand what your question might be about, then you may not get people to read and answer your question.
+
+Highlight what you believe to be the critical aspects of your question.  Finally, if you have trouble summarizing what aspects of your problem are important, you can try waiting until you have written your entire topic, including the {ref}`mwe` before trying to write the title.
+
+**Good titles**:
+- "Why doesn't pm.sample() use all my cores?"
+- "Multiprocessing BrokenPipeError: [Errno 32] Broken pipe"
+- "How I get az.plot_dist() to show priors?"
+
+**Not so good titles**:
+- "How do I do Bayesian inference?"
+- "Is there a better way to implement this model?"
+- "Hierarchical models"
+
+
+(ask_discourse/versions)=
+## Versions
+Please include information about what versions of the relevant packages you have installed and how they were installed, including:
+
+- PyMC version
+- Aesara version
+- Python version
+- System, etc. (e.g., linux, Mac, Windows, jupyter, docker, Google Colab)
+- How you installed PyMC (e.g., pip, conda, mamba)
+
+
+(ask_discourse/tags)
+## Tags
+Discourse includes two ways to classify topics: categories and tags.  Each topic belongs to exactly one category.  The most popular categories are for questions (e.g., `Questions/v4`, `Questions/v3`), but `Development` and `Sharing` are also quite popular.  Each topic may have more than one tag.  When you post your topic, search through the tags for any key words that might be relevant.  Tags include things like `bug`, `gaussian_process`, and `gpu`.
+
+Categories and tags are an important part of how discourse functions.  Users can subscribe to categories and/or tags that are of interest.  This means that careful selection of a category and one or more tags can alert the most relevant users to your question and get you answers quicker.
+
+```{important} Categories and tags are a critical part of how Discourse works.  Many users subscribe to categories and tags that are relevant to their own expertise.  This allows them to focus exclusively on questions that they are able to answer.  So if you post a questions under a generic category (e.g., "General") or post without any tags, your question is far less likely to attract the attention of those users most capable of answer your question!```
+```
```

Comment:
````suggestion
```{important}
Categories and tags are a critical part of how Discourse works.  Many users subscribe to categories and tags that are relevant to their own expertise.  This allows them to focus exclusively on questions that they are able to answer.  So if you post a questions under a generic category (e.g., "General") or post without any tags, your question is far less likely to attract the attention of those users most capable of answer your question!
```
````